pom.xml

```diff
@@ -688,6 +689,11 @@
                 <version>${springframework.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-web</artifactId>
+                <version>${springframework.version}</version>
+            </dependency>
```

Comment:
ShardingSphere is a backend project, please do not involve spring-web dependencies